### PR TITLE
for posterity

### PR DESCRIPTION
**Pull request by brettimus**
I spent much minutes (so time! wow.) cursing waterline's `add` method.

Why, you ask?

Well, if you attempt to add a record to a collection and that record is already in the collection, waterline throws an error. It's one of those things where someone you love does something and you still love them but why did they do that, you know?

My explanation above was kind of bad, though, so just [see this issue](https://github.com/balderdashy/waterline/issues/352) 

**tl;dr This one's for posterity.** Added a warning to the docs re: the above Issue

### CODE REVIEW

**Comment by rachaelshaw** [comment]
Thanks for adding that warning.
Would it be cool if we add you as a collaborator on this repo? All your PRs have been a huge help here

**Comment by brettimus** [comment]
DREAM. COME. TRUE. :joy_cat:

**Comment by brettimus** [comment]
(that's a yes. and it's my pleasure)
